controllers/validators/challenges.js

```diff
@@ -0,0 +1,19 @@
+
+'use strict';
+
+const validate = require('./validate-by-schema');
+
+module.exports = {
+  get: validate('getChallenge'),
+  getChallengesCommentedBy: validate('getChallengesCommentedBy'),
```

Comment:
Again, would it be perhaps worth having these generalized as `getDitsCommentedBy`? These schemata seem to not be ditType specific.